tests/utils/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@akashaorg/af-testing",
   "$schema": "../node_modules/nx/schemas/project-schema.json",
-  "sourceRoot": "tests/src",
+  "sourceRoot": "tests/utils",
```

Comment:
was it intentional to move the project.json file into utils?

Reply:
It is ... the tests folders will contain utils and later e2e.